business_requirement_deliverable_project_task_mandatory/models/project_project_category.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProjectProjectCategory(models.Model):
+    _inherit = 'project.project.category'
+
+    br_required = fields.Boolean(
+        'Business Requirement required',
+        help='If marked as true the task linked to the project'
```

Comment:
If marked as true the business requirement link in the task linked to this project will be mandatory